merlin/main.py

```diff
@@ -425,6 +425,7 @@ def process_server(args: Namespace):
     Route to the correct function based on the command
     given via the CLI
     """
+    os.environ["LC_ALL"] = "C"  # Necessary for Redis to configure LOCALE
```

Comment:
You may want to check this before setting, I believe there are other viable values.

Reply:
Maybe I should add a check that if it's set already we won't change it?